[2022-10-24T21:28:54.678+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: sales_datawarehouse.get_vendas manual__2022-10-24T21:28:50.432837+00:00 [queued]>
[2022-10-24T21:28:54.703+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: sales_datawarehouse.get_vendas manual__2022-10-24T21:28:50.432837+00:00 [queued]>
[2022-10-24T21:28:54.704+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-24T21:28:54.706+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-10-24T21:28:54.706+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-24T21:28:54.745+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): get_vendas> on 2022-10-24 21:28:50.432837+00:00
[2022-10-24T21:28:54.753+0000] {standard_task_runner.py:54} INFO - Started process 21511 to run task
[2022-10-24T21:28:54.760+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'sales_datawarehouse', 'get_vendas', 'manual__2022-10-24T21:28:50.432837+00:00', '--job-id', '369', '--raw', '--subdir', 'DAGS_FOLDER/sales_dw.py', '--cfg-path', '/tmp/tmpmk5_f9r6']
[2022-10-24T21:28:54.763+0000] {standard_task_runner.py:83} INFO - Job 369: Subtask get_vendas
[2022-10-24T21:28:54.765+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/sales_dw.py
[2022-10-24T21:28:55.662+0000] {task_command.py:384} INFO - Running <TaskInstance: sales_datawarehouse.get_vendas manual__2022-10-24T21:28:50.432837+00:00 [running]> on host 83cd0ab98720
[2022-10-24T21:28:56.281+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=pedromenegat
AIRFLOW_CTX_DAG_ID=sales_datawarehouse
AIRFLOW_CTX_TASK_ID=get_vendas
AIRFLOW_CTX_EXECUTION_DATE=2022-10-24T21:28:50.432837+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-24T21:28:50.432837+00:00
[2022-10-24T21:28:56.305+0000] {base.py:71} INFO - Using connection ID 'banco_vendas' for task execution.
[2022-10-24T21:28:58.666+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-10-24T21:28:58.691+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=sales_datawarehouse, task_id=get_vendas, execution_date=20221024T212850, start_date=20221024T212854, end_date=20221024T212858
[2022-10-24T21:28:58.729+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-10-24T21:28:58.771+0000] {local_task_job.py:273} INFO - 2 downstream tasks scheduled from follow-on schedule check
